src/index.ts

```diff
@@ -1 +1,2 @@
 export { atomWithValidate } from './atomWithValidate';
+export { validateAtom } from './validateForm';
```

Comment:
```suggestion
export { validateAtom } from './validateAtom';
```
Seems cleaner?

Reply:
semantically , there’s going to be more than one atom in most cases?

Reply:
Can be so. This is not a strong suggestion, so if it's just intentional (rather than a leftover), feel free to leave it as is.

Reply:
No, I get concern with the inconsistency it has with everything else but something that doesn't straight point to 1 atom would be a valid replacement.  

cause, if we name it `validateAtom` , instinctively people would just pass it an atom if not working in an IDE with language servers. 

`validateForm` is a placeholder but, also seems like there isn't a simple name for something that has a ValueGetter inside as it's different than most jotai API's

Reply:
I wouldn't usually export types and probably have it in a separate file if types are used in two files.

Actually, I don't mind much about the internal file names, which can be changed at any time. But, maybe we would revisit the API name `validateAtom`, because ValueGetter is a new one as you say.

Reply:
What if, we change the API to be like so 

```js 
const fNameAtom = atomWithValidate("", { validate: (v) => v });
const lNameAtom = atomWithValidate("", { validate: (v) => v });

validateAtomFamily(
  { firstName: fNameAtom, lastName: lNameAtom },
  { validate: (v) => v }
);
// or
validateAtomGroup(
  { firstName: fNameAtom, lastName: lNameAtom },
  { validate: (v) => v }
);


```

Reply:
"atom group" sounds better ("atom family" is too confusing with `atomFamily`).

```js
validateAtoms(async (get) => {
  const values = getAtomGroup(get)({ firstName: fNameAtom, lastName: lNameAtom });
  await validate(values);
});
```
can be my preference, but it might not be developer friendly.

Wait, we have `waitForAll` util.
```js
validateAtoms(
  { firstName: fNameAtom, lastName: lNameAtom }, // or [fNameAtom, lNameAtom]
  async (values) => ...
)
```
would match with it. Then, returning the combined atom values makes more sense. So, it's not just "validate"...

Hmm, let's think about this more.
```js
const fNameAtom = atomWithValidate("", { validate: (v) => v });
const lNameAtom = atomWithValidate("", { validate: (v) => v });

const combinedAtom = combineAtoms(
  { firstName: fNameAtom, lastName: lNameAtom },
  { validate: (v) => v }
);
```
It's almost like your proposition. Not sure if "combine" is the right word.

Reply:
I spend doing the same thing on paper 
and I think `validateAtomGroup` or `validateAtoms` and the API will be list the last comment 
```
```

Reply:
Sounds great. Now we are almost back to one of original propositions at some point. Would you update the implementation?

Reply:
Yeah, on it